getting-started/5_start_ui.md

```diff
@@ -0,0 +1,16 @@
+Back in our original terminal, we can now start the UI. Before we do so, we need to make a small adjustments for this environment.
+
+First, add the below line to `ui/.env`{{open}}
+
+<pre class="file" data-target="clipboard">DANGEROUSLY_DISABLE_HOST_CHECK=true
+</pre>
+
+Secondly, edit the websocket URL in `ui/src/config.ts`{{open}} to 
+
+<pre class="file" data-target="clipboard">wss://[[HOST_SUBDOMAIN]]-7575-[[KATACODA_HOST]].environments.katacoda.com/</pre>
+
+```
+yarn start
+```{{execute T1}}
+
+This starts the web UI connected to the running Sandbox and JSON API server. Once the web UI has been compiled and started, you should see `Compiled successfully!` in your terminal. You can now [open the UI tab](https://[[HOST_SUBDOMAIN]]-3000-[[KATACODA_HOST]].environments.katacoda.com). 
```

Comment:
I seem to manually have to click on “display port” here. Otherwise, it just seems to wait forever (or rather 30s since then I clicked on the button).

Reply:
The webserver does actually take a long time to start up. Did you wait until `Compiled successfully!`?

Reply:
Yes I waited until I saw `compiled successfully` before going here.